components/eamxx/cmake/ScreamUtils.cmake

```diff
@@ -45,40 +45,27 @@ set(SCREAM_CUT_TEST_MV_ARGS ${CUT_TEST_MV_ARGS})
 # Scream always excludes the ekat test session since it has its own
 list(REMOVE_ITEM SCREAM_CUT_EXEC_OPTIONS EXCLUDE_TEST_SESSION)
 
-# Libs are a position arg for SCREAM, not an optional arg like in EKAT
-list(REMOVE_ITEM SCREAM_CUT_EXEC_MV_ARGS LIBS)
-
 ###############################################################################
-function(CreateUnitTestExec exec_name test_srcs scream_libs)
+function(CreateUnitTestExec exec_name test_srcs)
 ###############################################################################
-  cmake_parse_arguments(cute "${SCREAM_CUT_EXEC_OPTIONS}" "${SCREAM_CUT_EXEC_1V_ARGS}" "${SCREAM_CUT_EXEC_MV_ARGS}" ${ARGN})
-  CheckMacroArgs(CreateUnitTestExec cute "${SCREAM_CUT_EXEC_OPTIONS}" "${SCREAM_CUT_EXEC_1V_ARGS}" "${SCREAM_CUT_EXEC_MV_ARGS}")
-
-  separate_cut_arguments(cute "${SCREAM_CUT_EXEC_OPTIONS}" "${SCREAM_CUT_EXEC_1V_ARGS}" "${SCREAM_CUT_EXEC_MV_ARGS}" options)
-
-  set(TEST_INCLUDE_DIRS
-    ${SCREAM_INCLUDE_DIRS}
-    ${CMAKE_CURRENT_SOURCE_DIR}
-    ${CMAKE_CURRENT_BINARY_DIR}
-  )
-
-  set(test_libs "${scream_libs};scream_test_support")
-  list(APPEND test_libs "${SCREAM_TPL_LIBRARIES}")
-
-  if (SCREAM_Fortran_FLAGS)
```

Comment:
This is not needed, since project-wide F flags are already attached to `scream_share`.